mitiq/zne/scaling/tests/test_folding.py

```diff
@@ -1245,6 +1319,41 @@ def test_fold_and_squash_max_stretch(fold_method):
     assert len(folded_with_squash_moments_not_specified) == 15
 
 
+@pytest.mark.parametrize(
+    "fold_method",
+    [
+        fold_global,
+    ],
+)
+def test_fold_global_no_squash(fold_method):
+    """Tests folding and squashing a two-qubit circuit."""
+    # Test circuit:
+    # 0: ───────H───────H───────H───────H───────H───
+    #
+    # 1: ───H───────H───────H───────H───────H───────
+
+    # Get the test circuit
+    d = 10
+    qreg = LineQubit.range(2)
+    circuit = Circuit()
+    for i in range(d):
+        circuit.insert(0, ops.H.on(qreg[i % 2]), strategy=InsertStrategy.NEW)
+
+    folded_not_squashed = fold_method(
+        circuit, scale_factor=3.0, squash_moments=False
+    )
+    folded_and_squashed = fold_method(
+        circuit, scale_factor=3.0, squash_moments=True
+    )
+    folded_with_squash_moments_not_specified = fold_method(
+        circuit, scale_factor=3.0
+    )  # Checks that the default is to squash moments
+
+    assert len(folded_not_squashed) == 30
+    assert len(folded_and_squashed) == 30
+    assert len(folded_with_squash_moments_not_specified) == 30
```

Comment:
Given that `squash_moments` is not an option anymore, maybe we can remove this test?

If you want to keep it, I would avoid the use of @pytest.mark.parametrize and replace all `fold_method` variables with `fold_global`.

Reply:
Initially, I had kept it in to make sure that Cirq's insert strategies would not change the moment structure when `base_circuit` was used directly. [EARLIEST](https://quantumai.google/cirq/circuits#insertstrategies) is the default but the way I understand it, it wants to minimize the moments if there's some space in another moment. 

With the new code, the moment structure will be maintained throughout the folding process. Can you think of a case when insert strategy would cause issues here ? I don't understand them beyond what's provided in the documentation.

Reply:
@purva-thakre,  `InsertStrategy` is a Cirq option that only acts during the initial construction of the input circuit but has no effects later. In principle one could manually define the input circuit moment by moment and get the same result. 

So I propose to remove this test since the `squash_moments` option no longer exists for `fold_global`.